lib/components/util/formatted-day-of-week.js

```diff
@@ -0,0 +1,37 @@
+import { FormattedMessage } from 'react-intl'
+import PropTypes from 'prop-types'
+
+/**
+ * Returns a FormattedMessage component for the compact day of week such that i18n IDs are
+ * hardcoded and can be kept track of by format.js tools
+ * TODO: More general FormattedDayOfWeek component that accepts a type of DayOfWeek
+ * 'compact', or 'plural', etc. ?
+ */
+export default function FormattedDayOfWeek ({day}) {
+  switch (day) {
```

Comment:
Like the FormattedDayOfWeekCompact, using PropTypes to validate the day (https://stackoverflow.com/a/60912152) and then generating the IDs using template strings could make this component much cleaner.